src/main/java/picard/sam/CreateSequenceDictionary.java

```diff
@@ -196,4 +214,59 @@ private String md5Hash(final byte[] bytes) {
         }
         return s;
     }
+
+    private SortingCollection<String> makeSortingCollection() {
+        final String parent = System.getProperty("java.io.tmpdir") + "/" + System.getProperty("user.name");
+        final String child = getClass().getSimpleName();
+        final File tmpDir = new File(parent, child);
+        IOUtil.deleteDirectoryTree(tmpDir);
```

Comment:
deleting random directory trees on the path is not a nice way to behave....please only delete what you create, after you create it, and be sure that it didn't exist before creating it.....